src/FlakyCommand.php

```
@@ -32,37 +33,40 @@ public function __construct(Command $command)
         $this->varyOnInput = $keys;
 
         return $this;
     }
 
-    public function instance()
+    public function instance(): Flaky
     {
         return Flaky::make($this->generateCommandId())
             // Only enable protection if the command is running
             // via schedule not directly from CLI.
             ->disableFlakyProtection($disabled = !$this->isScheduledCommand());
```

PHPMD (2.15.0)

Avoid unused local variables such as '$disabled'. (undefined)

(UnusedLocalVariable)